### PR TITLE
Added some projects

### PR DESCRIPTION
**Pull request by hbakouane**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -96,6 +96,7 @@
   - [Yassine Bennkhay](https://yassine-bennkhay.tech/) - Flutter, Dart,...
   - [Yasser Tahiri](https://blog.yezz.me/) - Security, Python, APIs, ....
   - [Mouhcine Toumi](https://blog.mouhcinetoumi.com/) - Data Engineering, branding, self dev...
+  - [Haytam Bakouane](http://hbakouane.dev/) - Laravel/Vue developer
```
Comment (JefferyHus) [inline readme.md:99]: Just make it `https` here and we are good to go

Comment (hbakouane) [reply readme.md]: Done

**Review by yezz123 — APPROVED** [review]
LGTM!

**Comment by yjose** [comment]
@hbakouane @JefferyHus @yezz123 i just deleted @hbakouane portfolio link from the blogs section as we don't accept portfolios but blogs instead.
@JefferyHus @yezz123  can we please make sure we respect this rule in the upcoming reviews 🙏

**Comment by aboullaite** [comment]
My bad! didn't check it before merging. good catch @yjose

**Comment by hbakouane** [comment]
Sorry guys, but I do have a blog on Medium if that's acceptable: https://hbakouane.medium.com, Thanks tho

**Pull request merged**